test/compilable/dtoh_functions.d

```diff
@@ -24,6 +27,56 @@ extern int32_t baz2(int32_t x);
 extern int32_t baz4(int32_t x = 42);
 
 extern size_t baz5(size_t x = 42);
+
+enum class E : int64_t
+{
+    m = 1LL,
+};
+
+extern void enums(uint64_t e = $?:32=1LLU|64=E::m$);
```

Comment:
THis is necessary because the frontend generates different AST's for `-m32` (`IntegerExp`) and `-m64` (`DotIdExp`) ...